src/redux/sagas/motions/moveFundsMotion.ts

```diff
@@ -112,65 +97,197 @@ function* moveFundsMotion({
+            isMultiSig: true,
+          },
+        );
+
+        const rootDomain = colonyDomains.find((domain) =>
+          BigNumber.from(domain.nativeId).eq(Id.RootDomain),
+        );
+
+        if (!rootDomain) {
+          throw new Error('Cannot find rootDomain in colony domains');
+        }
+
+        const fromChildSkillIndex = yield call(getChildIndexLocal, {
+          networkClient: colonyClient.networkClient,
+          parentDomainNativeId: rootDomain.nativeId,
+          parentDomainSkillId: rootDomain.nativeSkillId,
+          domainNativeId: fromDomain.nativeId,
+          domainSkillId: fromDomain.nativeSkillId,
+        });
+
+        const toChildSkillIndex = yield call(getChildIndexLocal, {
+          networkClient: colonyClient.networkClient,
+          parentDomainNativeId: rootDomain.nativeId,
+          parentDomainSkillId: rootDomain.nativeSkillId,
+          domainNativeId: toDomain.nativeId,
+          domainSkillId: toDomain.nativeSkillId,
+        });
+
+        const encodedAction = colonyClient.interface.encodeFunctionData(
+          contractMethod,
+          [
+            ...(isOldVersion
+              ? []
+              : [multiSigPermissionDomainId, multiSigChildSkillIndex]),
+            userPermissionDomainId,
+            fromChildSkillIndex,
+            toChildSkillIndex,
+            fromPot,
+            toPot,
+            amount,
+            tokenAddress,
+          ],
+        );
+
+        return {
+          context: ClientType.MultisigPermissionsClient,
+          methodName: 'createMotion',
+          identifier: colonyAddress,
+          params: [
+            userPermissionDomainId,
+            userChildSkillIndex,
+            [colonyAddress],
+            [encodedAction],
+          ],
+          group: {
+            key: batchKey,
+            id: metaId,
+            index: 0,
+          },
+          ready: false,
+        };
+      }
+
+      const votingReputationClient = yield colonyClient.getExtensionClient(
+        Extension.VotingReputation,
+      );
+
+      const motionChildSkillIndex = yield call(
+        getChildIndex,
+        colonyClient.networkClient,
+        colonyClient,
+        createdInDomain.nativeId,
+        fromDomain.nativeId,
+      );
+
+      const { skillId } = yield call(
+        [colonyClient, colonyClient.getDomain],
+        createdInDomain.nativeId,
+      );
+
+      const { key, value, branchMask, siblings } = yield call(
+        colonyClient.getReputation,
+        skillId,
         AddressZero,
-        encodedAction,
-        key,
-        value,
-        branchMask,
-        siblings,
-      ],
-      group: {
-        key: batchKey,
-        id: metaId,
-        index: 0,
-      },
-      ready: false,
-    });
+      );
+
+      const [fromPermissionDomainId] = yield call(
+        getPermissionProofs,
+        colonyClient.networkClient,
+        colonyClient,
+        fromDomain.nativeId,
+        [ColonyRole.Funding],
```

Comment:
This can use `const requiredRoles = PERMISSIONS_NEEDED_FOR_ACTION.TransferFunds;`